src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,43 @@
+package duke;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a Deadline Task.
+ *
+ * @author Denzel Tan
+ */
+public class Deadline extends Task {
+    protected LocalDate date;
+
+    /**
+     * Constructor for the deadline type of task.
+     *
+     * @param description description of the deadline task
+     * @param date the deadline timing of the deadline task
+     */
+    public Deadline(String description, LocalDate date) {
+        super(description);
+        this.date = date;
+    }
+
+    /**
+     * Show the date of the deadline in a string.
+     *
+     * @return the string representation of the date of the deadline
+     */
+    public String deadlineDateStr() {
+        return date.getMonth() + " " + date.getDayOfMonth() + " " + date.getYear();
+    }
```

Comment:
Should the name of this method be a verb?
```suggestion
    public String getDeadlineDateStr() {
        return date.getMonth() + " " + date.getDayOfMonth() + " " + date.getYear();
    }
```